api/tests/Api/ContentNodes/ChecklistNode/UpdateChecklistNodeTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Tests\Api\ContentNodes\ChecklistNode;
+
+use App\Entity\ContentNode\ChecklistNode;
+use App\Tests\Api\ContentNodes\UpdateContentNodeTestCase;
+
+/**
+ * @internal
+ */
+class UpdateChecklistNodeTest extends UpdateContentNodeTestCase {
+    public function setUp(): void {
+        parent::setUp();
+
+        $this->endpoint = '/content_node/checklist_nodes';
+        $this->defaultEntity = static::getFixture('checklistNode1');
+    }
+
+    public function testAddChecklistItemIsDeniedForGuest() {
+        $checklistItemId = static::getFixture('checklistItem1_1_2')->getId();
+        static::createClientWithCredentials(['email' => static::getFixture('user3guest')->getEmail()])
+            ->request('PATCH', $this->endpoint.'/'.$this->defaultEntity->getId(), ['json' => [
+                'addChecklistItemIds' => [$checklistItemId],
+            ], 'headers' => ['Content-Type' => 'application/merge-patch+json']])
+        ;
+        $this->assertResponseStatusCodeSame(403);
+        $this->assertJsonContains([
+            'title' => 'An error occurred',
+            'detail' => 'Access Denied.',
+        ]);
+    }
+
+    public function testAddChecklistItemIsDeniedForMember() {
```

Comment:
```suggestion
    public function testAddChecklistItemForMember() {
```
does not look denied, and should not be denied